### PR TITLE
Add page for game servers under 'Other Services'.

### PR DESCRIPTION
**Pull request by AndyJenks**
Hopefully I have inferred correctly the proper way to do things. Certainly it seemed to work for me locally.

### CODE REVIEW

File: source/services/gameserver.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Game servers
+------------
+
+If you want to run a server for your favourite online game (or any other
+long-running application) you should use ``userservers.srcf.net``, as
```
Comment (CHTJonas) [inline source/services/gameserver.rst:5]:
```suggestion
long-running application) you should use ``doom.srcf.net``, as 
```

**Review by CHTJonas — CHANGES_REQUESTED** [review]
Looks good! :tada:

One small thing: as per #35 I think it's been decided to retire the `userservers.srcf.net` hostname because it's a CNAME for `cavein`, which has now been decommissioned; switching it over to point at `doom` would cause SSH host key mismatch warnings so best to have people SSH to `doom` directly.

**Review by doismellburning — APPROVED** [review]

**Review by CHTJonas — APPROVED** [review]
The sysadmins may wish to comment on the "1GB of RAM" part specifically but LGTM! 🚀

**Review by rsa33 — APPROVED** [review]
There's apparently also a `nogui` option (or `--nogui`?), but in the interests of keeping the example broader (and because hopefully MC knows not to try and launch a GUI for users without a graphical session), let's keep it as-is.

**Pull request merged**